docs/installation.md

```diff
@@ -204,9 +246,53 @@ plugins_infos:
   2024-01-17T01:01:46-06:00 INF Executor send request to cpu_monitor module=executor
   2024-01-17T01:01:46-06:00 INF response: SUCCESS module=executor
   ```
+  You can set your own path for config file such as 
+  ```
+  ~$ ./vatz start --config <paht_to_your_own_file_at/config_file.yaml>
+  ```
 
+### 5.3. Notification
+Alert notifications are sent to the configured channels when issues are detected in the monitored node. The specific alert conditions depend on each plugin.
 
-### 3. Notification
-The alert notification will be sent to configured channels if there are problems in the monitored node. The alert conditions vary for each plugin. 
+> Dispatcher channels can be configured to operate in two modes
+> - **Non-subscription mode**
+> - **Subscription-based mode**
+
+Below are examples of both modes, along with the list of installed plugins:
+```
+ vatz plugin list                                                                                                                                                             
++-------------------+------------+---------------------+-----------------------------------------------------------------+---------+
+|     NAME          | IS ENABLED | INSTALL DATE        | REPOSITORY                                                      | VERSION |
++-------------------+------------+---------------------+-----------------------------------------------------------------+---------+
+| cosmos_watcher    | true       | 2024-08-28 18:50:38 | github.com/dsrvlabs/vatz-plugin-watchers/plugins/watcher_cosmos | latest  |
+| shentu_watcher    | true       | 2024-08-28 18:50:44 | github.com/dsrvlabs/vatz-plugin-watchers/plugins/watcher_cosmos | latest  |
+| osmosis_watcher   | true       | 2024-08-28 18:50:44 | github.com/dsrvlabs/vatz-plugin-watchers/plugins/watcher_cosmos | latest  |
++-------------------+------------+---------------------+-----------------------------------------------------------------+---------+
+```
 
 
+1. **Non-subscription mode**: In this mode, notifications are automatically delivered from any declared plugin that meets its own alert conditions. All registered dispatcher channels will send alert messages from all plugins.
+```
+    dispatch_channels:
+      - channel: "discord"
+        secret: "https://discord.com/api/webhooks/221449818154281/7PfPpuWv4uK4wkPp-uWT1nJalAesD0YgSZA2j2EL7YvAN1ah32"
+        subscriptions:
+      - channel: "pagerduty"
+        secret: "y_NbAkKc66ryYTWUX4YEu801s"
+```
+  In this mode, there is no need to set any specific subscriptions. If any plugin’s alert conditions are met, the alert messages will be sent to both Discord and PagerDuty.
+
+2. **Subscription-based mode**: This mode allows you to subscribe to specific plugins, ensuring that you receive notifications only from the plugins you have explicitly subscribed to. Only the subscribed plugins’ alert messages will be delivered through the dispatcher channel.
```

Comment:
just for  clarity
`This mode allows you to subscribe to specific plugins, ensuring that you receive notifications only from those you select. In this case, only alert messages from your subscribed plugins will be delivered through the dispatcher channel.`

Reply:
updated!